include/dxc/DXIL/DxilModule.h

```diff
@@ -333,30 +333,28 @@ class DxilModule {
   std::vector<std::unique_ptr<DxilSampler> > m_Samplers;
 
   // Geometry shader.
-  DXIL::PrimitiveTopology m_StreamPrimitiveTopology;
-  unsigned m_ActiveStreamMask;
+  DXIL::PrimitiveTopology m_StreamPrimitiveTopology = DXIL::PrimitiveTopology::Undefined;
+  unsigned m_ActiveStreamMask = 0;
 
-private:
   enum IntermediateFlags : uint32_t {
     LegacyResourceReservation = 1 << 0,
   };
 
-private:
   llvm::LLVMContext &m_Ctx;
-  llvm::Module *m_pModule;
-  llvm::Function *m_pEntryFunc;
-  std::string m_EntryName;
-  std::unique_ptr<DxilMDHelper> m_pMDHelper;
-  std::unique_ptr<llvm::DebugInfoFinder> m_pDebugInfoFinder;
-  const ShaderModel *m_pSM;
-  unsigned m_DxilMajor;
-  unsigned m_DxilMinor;
-  unsigned m_ValMajor;
-  unsigned m_ValMinor;
-  bool m_ForceZeroStoreLifetimes;
+  llvm::Module *m_pModule = nullptr;
+  llvm::Function *m_pEntryFunc = nullptr;
+  std::string m_EntryName = "";
```

Comment:
I'd suggest leaving the STL containers alone, they have default constructors that already initialize themselves.

Reply:
In fact, I think the `= nullptr;` on the `llvm::DebugInfoFinder` is what's breaking the CI build.